Adapter/Adapter_AdvancedPaymentOptionsConverter.php

```diff
@@ -17,7 +17,14 @@ public function getPaymentOptions()
             $advancedPaymentOptions = [];
         }
 
-        $paymentOptions = array_merge($displayPaymentEUOptions, $advancedPaymentOptions);
+
```

Comment:
Could you please add a TODO here so that we don't forget to add the compatibility layer with the already adapted modules?